### ISSUE TITLE
Different slices at last volume when manually stopping

### ISSUE THREAD

**Issue by karl-koschutnig** [CMRR-C2P/MB#386]
Dear Expert, 
We use the CMRR-bold sequence on our Siemens VIDA (XA 50). We have many studies with dynamic acquisition duration. This means the number of volumes depends on the subject's response time, so we can not set a specific number of volumes in the sequence. Hence, when we manually cancel the acquisition, the last volume is not entirely measured; it has a different number of slices. So, I was wondering if there is an option to measure the complete last volume or to discard it.
Our pipeline that automatically converts DCM into BIDS is having trouble when the number of slices is not the same for all volumes. 

Thanks,
Karl

**Comment by eauerbach** [comment]
No, there is no option to address this at the sequence level; the solution would have to be for your pipeline to disregard the last volume
